blocks/identity-block/components/SocialSignOn/index.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { useEffect, useState } from 'react';
+import useIdentity from '../Identity';
+import './styles.scss';
+
+const SocialSignOn = ({ onError, redirectURL }) => {
+  const { Identity } = useIdentity();
+  const config = Identity?.configOptions ?? {};
+
+  const [isGoogleInitialized, setIsGoogleInitialized] = useState(false);
+  const [isFacebookInitialized, setIsFacebookInitialized] = useState(false);
+
+  if (!window.onFacebookSignOn) {
```

Comment:
This should also use the Engine theme SDK `isServerSide` method to stop server side render errors

Reply:
This has been applied.